.gitignore

```diff
@@ -239,11 +239,6 @@ pip-log.txt
 ##########
 .idea/
 
-##########
-# Visual Studio Code
-##########
-.vscode/
```

Comment:
Any particular reason for removing the `.vscode/` ignore?

Reply:
Because we don't want to ignore it (it's already checked in)

Reply:
I think we should keep `.vscode/` ignored, as files created by vscode in there may contain platform and computer specific settings.

Any files that we want committed in there, should be force checked in.

This is to avoid accidentally committing local config changes doing silly stuff like `git add -A` or `git commit -a`.